SecretManagement.LastPass.Extension/SecretManagement.LastPass.Extension.psm1

```diff
@@ -195,3 +289,59 @@ function Test-SecretVault
     $status = Invoke-lpass 'status' -ErrorAction SilentlyContinue
     return ($status -match "Logged in as .*")
 }
+
+function Get-SimpleSecret {
+    [CmdletBinding()]
+    param (
+        $Fields,
+        $Note
+    )
+    
+    $IsCredentials = $Fields.where( { $_.key -in @('Username', 'Password') }, 'first').count -eq 1
+    if ($IsCredentials) {
+        $username = $Fields.Where( { $_.key -eq 'Username' }, 'first') | Select-Object -ExpandProperty value
+        if ($null -eq $username) { $username = '' }
+        $password = $Fields.Where( { $_.key -eq 'Password' }, 'first') | Select-Object -ExpandProperty value
+        if ($null -eq $password) { $password = '' }
+        if ("" -ne $password) { $password = $password | ConvertTo-SecureString -AsPlainText -Force }
+        $output = [System.Management.Automation.PSCredential]::new($username, $password)
+    }
+    else {
+        if ($null -eq $Note) { $output = "" } else { $output = $Note }
+    }
+    return $output
```

Comment:
another possible implementation I think:

```suggestion
    $username = $Fields.Where( { $_.key -eq 'Username' }, 'first') | Select-Object -ExpandProperty value
    $password = $Fields.Where( { $_.key -eq 'Password' }, 'first') | Select-Object -ExpandProperty value
    if ($username -or $password) {
        if ($null -eq $username) { $username = '' }
        if ($null -eq $password) { $password = '' }
        if ("" -ne $password) { $password = $password | ConvertTo-SecureString -AsPlainText -Force }
        return [System.Management.Automation.PSCredential]::new($username, $password)
    }

    if ($null -ne $Note) {
        return $Note
    }

    return ''
```

Reply:
I like it. 
It's cleaner.
(Note: I didn't notice the "Commit suggestion" button until after committing manually. I'll know that for next time)